executor/point_get_test.go

```diff
@@ -593,3 +593,39 @@ func (s *testPointGetSuite) TestClusterIndexCBOPointGet(c *C) {
 		res.Check(testkit.Rows(output[i].Res...))
 	}
 }
+
+func (s *testPointGetSuite) TesPointGetReadLock(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("create table point (id int primary key, c int, d varchar(10), unique c_d (c, d))")
+	tk.MustExec("insert point values (1, 1, 'a')")
+	tk.MustExec("insert point values (2, 2, 'b')")
+	tk.MustExec("alter table point read")
+	result, err := tk.Exec("select id as ident from point where id = 1")
+	c.Assert(err, IsNil)
+	fields := result.Fields()
+	c.Assert(fields[0].ColumnAsName.O, Equals, "ident")
+	result.Close()
+	tk.MustQuery(`select * from point where id = 1;`).Check(testkit.Rows(
+		`1 1 'a'`,
+	))
+	tk.MustExec("unlock tables")
+}
+
+func (s *testPointGetSuite) TesPointGetWriteLock(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("create table point (id int primary key, c int, d varchar(10), unique c_d (c, d))")
+	tk.MustExec("insert point values (1, 1, 'a')")
+	tk.MustExec("insert point values (2, 2, 'b')")
+	tk.MustExec("alter table point write")
```

Comment:
ditto.

`lock tables point write`

Reply:
Also need to make sure the config of `EnableTableLock` is `true`, otherwise the table lock will not take effect.

Reply:
> ditto.
> 
> `lock tables point write`

fix.